src/platform/lumin-runtime/elements/builders/video-builder.js

```diff
@@ -42,9 +43,11 @@ export class VideoBuilder extends QuadBuilder {
         const element = this._callNodeFunction(prism, 'createVideoNode', width, height);
 
         if (videoPath !== undefined) {
-            this._callNodeAction(element, 'setVideoPath', videoPath);
-        } else {
-            this._callNodeAction(element, 'setVideoUri', videoUri);
+          if (isUrl(videoPath)) {
+            this._callNodeAction(element, "setVideoUri", videoPath);
+          } else {
+            this._callNodeAction(element, "setVideoPath", videoPath);
+          }
```

Comment:
I think it's not enough to update `videoUri` in **create** function only. What about **update** function? Can we get rid of `videoPath` and `videoUri` and support only one property (visible from JSX)?

Reply:
@SPaluchiewiczSofomo and @pleszkiewicz, please review the refactored code.